src/pages/Debug/Report/DebugReportPage.tsx

```diff
@@ -1,35 +1,107 @@
 import type {DebugParamList} from '@libs/Navigation/types';
+import {getAllReportErrors} from '@libs/OptionsListUtils';
 import DebugDetails from '@pages/Debug/DebugDetails';
 import DebugJSON from '@pages/Debug/DebugJSON';
 import Debug from '@userActions/Debug';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
 import type SCREENS from '@src/SCREENS';
+import {isEmptyObject} from '@src/types/utils/EmptyObject';
 import DebugReportActions from './DebugReportActions';
 
 type DebugReportPageProps = StackScreenProps<DebugParamList, typeof SCREENS.DEBUG.REPORT>;
 
+type Metadata = {
+    title: string;
+    subtitle: string;
+    message?: string;
+    action?: {
+        name: string;
+        callback: () => void;
+    };
+};
+
 function DebugReportPage({
     route: {
         params: {reportID},
     },
 }: DebugReportPageProps) {
     const {translate} = useLocalize();
     const styles = useThemeStyles();
+    const StyleUtils = useStyleUtils();
+    const theme = useTheme();
     const [report] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT}${reportID}`);
+    const [reportActions] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportID}`);
+
+    const metadata = useMemo<Metadata[]>(() => {
+        if (!report) {
+            return [];
+        }
+
+        const reasonLHN = DebugUtils.getReasonForShowingRowInLHN(report);
+        const reasonGBR = DebugUtils.getReasonForShowingGreenDotInLHNRow(report);
+        const reportActionGBR = DebugUtils.getGBRReportAction(report, reportActions);
+        const reportActionRBR = DebugUtils.getRBRReportAction(report, reportActions);
+
+        return [
+            {
+                title: translate('debug.visibleInLHN'),
+                subtitle: translate(`debug.${!!reasonLHN}`),
+                message: reasonLHN ? translate(reasonLHN) : undefined,
+            },
+            {
+                title: translate('debug.GBR'),
+                subtitle: translate(`debug.${!!reasonGBR}`),
+                message: reasonGBR ? translate(reasonGBR) : undefined,
+                action: reportActionGBR
+                    ? {
+                          name: translate('common.view'),
+                          callback: () =>
+                              Navigation.navigate(
+                                  ROUTES.REPORT_WITH_ID.getRoute(
+                                      reportActionGBR.childReportID ?? reportActionGBR.parentReportID ?? report.reportID,
+                                      reportActionGBR.childReportID ? undefined : reportActionGBR.reportActionID,
+                                  ),
+                              ),
+                      }
+                    : undefined,
+            },
+            {
+                title: translate('debug.RBR'),
+                subtitle: translate(`debug.${!isEmptyObject(getAllReportErrors(report, reportActions))}`),
```

Comment:
Let's use the same condition in LHN

https://github.com/Expensify/App/blob/86360745431edb355c91e5594c741a54e5f88581/src/components/LHNOptionsList/OptionRowLHN.tsx#L116

Reply:
`optionItem.brickRoadIndicator` is created specially of LHN in the same way that I implemented on `DebugReportPage`, so I'm using the same condition.

Here it is the implementation of `optionItem.brickRoadIndicator`:

<img width="1141" alt="Screenshot 2024-10-01 at 16 31 44" src="https://github.com/user-attachments/assets/03c9cf91-7be8-4046-a1f0-8f5710e3d0f4">